paddle/fluid/framework/executor.h

```diff
@@ -44,17 +44,11 @@ class Executor {
 
   explicit Executor(const platform::Place& place);
 
-#ifdef PADDLE_WITH_DISTRIBUTE
   /*
-   * Sending signal to pserver to mark current pass started.
+   * Close this Executor.
+   * Calling this method will send complete messages to all pserver instances.
    */
-  void BeginPass();
-
-  /*
-   * Sending signal to pserver to mark current pass finished.
-   */
-  void EndPass();
-#endif
+  void Close();
```

Comment:
Why changed to `Close`? The underlying method is still called `Complete`

Reply:
complete is a subsetof close(). close can also clean up other resources in the future. It's similar to tf's ```Session.close```. It cleans up all related resources (such as complelte) and shutdown the session.

Reply:
I see

Reply:
I see